### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by Raikouu**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -356,152 +356,152 @@
...
-    <string name="quest_group">Group %s</string>
-    <string name="quest_label_objective">Objective</string>
+    <string name="quest_group">Gruppe %s</string>
+    <string name="quest_label_objective">Ziel</string>
 
     <!-- ################ -->
     <!-- ## KINSECT MENU ## -->
     <!-- ################ -->
 
     <string name="title_kinsects">Kinsects</string>
```
Comment (Raikouu) [inline app/src/main/res/values/strings.xml:408]: Forgot to change Kinsects to Kinstekten

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -356,152 +356,152 @@
...
+    <string name="user_equipment_set_deleted">%1$s wurde gelöscht</string>
+    <string name="user_equipment_new_set_name">Set bennen</string>
+    <string name="user_equipment_apply">Anwenden</string>
+    <string name="user_equipment_cancel">Abbrechen</string>
 
     <string name="user_equipment_set_filter_name">Name</string>
     <string name="user_equipment_set_filter_slot_level">Slot Level</string>
-    <string name="user_equipment_set_filter_elemental_defense">Elemental Defense</string>
-    <string name="user_equipment_set_filter_add_skill">Add a new skill</string>
+    <string name="user_equipment_set_filter_elemental_defense">Elementale Verteidung</string>
```
Comment (Raikouu) [inline app/src/main/res/values/strings.xml:494]: Typo here should be Elementale Verteidigung

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -356,152 +356,152 @@
...
     <string name="about_body">
-        Gathering Hall Studios is an open source team of designers and developers
-        dedicated to building high-quality apps for the Monster Hunter series.
-        \n\nIf you’d like to join or contribute, we’re always open.
-        Contact us at contact@gatheringhallstudios.com or visit our Discord.
-        You can also visit our Github to view the source code the app data projects.
+        Gathering Hall Studios ist ein open source Team von Designern und Developern,
+        welche der Erstellung von high-quality Apps für die Monster Hunter Serie gewidmet sind.
+        \n\nFalls du uns beitreten oder helfen willst sind wir immer offen.
+        Kontact uns auf contact@gatheringhallstudios.com oder besuche unsern Discord-Server.
```
Comment (Raikouu) [inline app/src/main/res/values/strings.xml:504]: Typo here should be Kontaktiere uns auf

**Comment by jaysondc** [comment]
Integrated your changes manually. Thanks for the contribution!

**Pull request closed without merging**